platform/ui/src/components/ohifModal/OHIFModal.styl

```diff
@@ -0,0 +1,56 @@
+.OHIFModal
+  background-color: var(--ui-gray-darker)
+  border-color: var(--ui-border-color)
+  color: var(--text-secondary-color)
+  border-radius: 6px
+  border: 0
+  color: var(--text-primary-color)
+  position: relative
+  -webkit-box-shadow: 0 3px 9px rgba(0,0,0,.5);
+  box-shadow: 0 3px 9px rgba(0,0,0,.5);
+  background-clip: padding-box;
+  outline: 0;
+
+  @media (min-width: 768px)
+    width: 600px;
```

Comment:
May be worth setting a percentage width here and a `min-width`

Reply:
I'm using the same media from modal.styl. Honestly, I find percentages to be harder to define, do you have something in mind?

Reply:
I only suggested percentage so that this breakpoint would continue to work as we approach the lower mobile threshold of ~320px. You could do that with `width: 78%; min-width: 300px`

While we're not there yet, I could see a Modal becoming an "Action Sheet" or something similar at mobile resolutions; this would just buy us a bit of grace until we get there / while we transition.